### PR TITLE
leaderboard setup

### PR DESCRIPTION
**Pull request by Samuelwanza**
I have conducted the following:
-Setup the web pack for the leaderboard project
-Setup the HTML template for the project
-Styled the layout of the leaderboard in styles.css
-used git-flow for version control
-used import es6 module

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,149 @@
...
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">Linters</a></li>
+    <li><a href="https://reactjs.org/">Javascript</a></li>
```
Comment (Efiamotu-1) [inline README.md]: - [ ] Please provide the appropriate links to each technology. you have used the same link which is the link to `react` for each tech. I kindly suggest you provide the correct link so your readme looks professional :+1: :sparkles:

File: README.md [hunk 2]
```diff
@@ -1,2 +1,149 @@
...
+- Code Editor
+- Git
+
+### Setup
+
+To get started, clone this repository to your desired folder:
+
+```shell
+cd my-folder
+git clone https://github.com/Samuelwanza/todolist/.git
```
Comment (Efiamotu-1) [inline README.md]: - [ ] You are still making use of the repo for your todolist project. please kindly change this to leaderboard so users are directly to the correct repo :+1:

File: README.md [hunk 3]
```diff
@@ -1,2 +1,149 @@
...
+3. Then run npm run start to spawn the development server
+4. This will launch the project in your default web browser and automatically refresh it when changes are made.
+5. Verify that the tests are running successfully and check the console for any error messages or test results.
+
+By following these steps, you can effectively run and validate the tests for the project using Live Server in VSCode.
+
+### Deployment:
+
+Deployment date: 10th June, 2023.
+[Deployed Link](https://samuelwanza.github.io/todolist/dist/)
```
Comment (Efiamotu-1) [inline README.md]: - [ ] You are still making use of the deployed link for your todolist project. please kindly change this to leaderboard so users are directly to the correct deployed version :+1:

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Samuelwanza , 

# STATUS: Changes Required

## Highlights :dart: 
* Correct Gitflow :heavy_check_mark: 
* Good job setting the project up correctly :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR Title & summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @Samuelwanza,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] Great job on the functionality for the setup🔥
- [ ] Followed Gitflow 🔥
- [ ] No Linters error ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**